src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspacesNetCore.cs

```diff
@@ -40,7 +40,7 @@ public override void MetadataReference()
             VisualStudio.SolutionExplorer.EditProjectFile(project);
             VisualStudio.Editor.SetText(@"<Project Sdk=""Microsoft.NET.Sdk"">
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Have we tried this?

```suggestion
    <TargetFramework>net461</TargetFramework>
    <UseWPF>true</UseWPF>
```

Reply:
Oh, right, that would change because we need the reference to not exist initially, and then be added.

This test really needs to be rewritten to use `BigInteger` instead of `System.Windows.Point`. The WPF dependency isn't correctly handled by our features.

Reply:
Yep I tried <UseWPF> and it doens't work.
For using https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/, solution explorer is complaining it can't find WindowsBase dll
And so I try to switch back to net46 to see if it works..

Reply:
Maybe `BigInteger` is not the best choice because the default template is referecing System.Numerics by default
![image](https://user-images.githubusercontent.com/24360909/149588697-67dec300-1509-4b75-b050-a0d45569246e.png)
We may want to find a struct in other assemblies

Reply:
Yes, we'll want to find a different assembly. I was thinking System.Numerics wasn't included by default.